packages/vanilla-js/src/AuthNClient/index.ts

```diff
@@ -155,6 +155,10 @@ export class AuthNClient {
       version_ = `${version}${!!version ? "/" : ""}`;
     }
 
+    if (!!this.config.usePathApi) {
+      return `${protocol}://${this.config.domain}/api/authn/${version_}${endpoint}`;
```

Comment:
So this currently still requires a domain and protocol, the sdks went for a template approach, https://github.com/pangeacyber/pangea-python/pull/654/files, https://github.com/pangeacyber/pangea-javascript/pull/2071,  I was thinking maybe the authn client component should follow similar suit.

The react-auth lib will have to get updated as well?

Reply:
Templating would be good from an external user perspective, and we can add it for that case. The domain is just a hostname, so unless we want to support insecure use on a domain name, this should work.

Reply:
Ya I guess I see the localhost logic for changing the port too, which is part of the plan to support port forwarding and running on localhost,

So the react-auth making the `https://authn.dev.aws.pangea.cloud/v2/client/token/check` call would just work? How does it know to switch to `api/authn/v2/client/token/check`?

Reply:
React-auth doesn't use the client directly, it calls a method that is passed to it, which calls the client in the AuthN hosted page code. In the hosted page, the `usePathApi` flag is set to true for private cloud, so the client knows to construct the url in the right way.

Reply:
```
const { success, response } = await client.userinfo(code);

      if (success) {
        const token = response.result?.active_token?.token;

        if (token) {
          // if using an opaque token or has a valid JWT
          if (!config?.useJwt || (config?.useJwt && (await validate(token)))) {
            processLogin(response);
          }
        } else {
          const msg = "Missing Token";
          setError(msg);
        }
      } else {
        setError(response.summary);
      }
      storageAPI.removeItem(STATE_DATA_KEY);
      setLoading(false);
```